packages/fxa-content-server/package.json

```diff
@@ -62,7 +62,7 @@
     "fxa-common-password-list": "0.0.2",
     "fxa-crypto-relier": "2.3.0",
     "fxa-geodb": "1.0.4",
-    "fxa-js-client": "1.0.12",
+    "fxa-js-client": "^1.0.13",
```

Comment:
Unexpected caret dependency version here?

Reply:
Yup, we've decided to start using carrots.